src/components/Icon/tests/Icon.test.tsx

```diff
@@ -22,7 +27,7 @@ describe('<Icon />', () => {
 
     it('renders a React Element when source is given a React Stateless Functional Component', () => {
       const element = mountWithApp(<Icon source={PlusMinor} />);
-      expect(element).toContainReactComponentTimes(PlusMinor, 1);
+      expect(element).toHaveReactProps({source: PlusMinor});
```

Comment:
I think the original line is the right intent. `expect(element).toHaveReactProps` asserts what we passed into the element (i.e. what we did on the line above), but doesn't test anything about the internals of Icon